src/main/java/ch/heigvd/amt/stack/application/GamificationFacade.java

```diff
@@ -45,6 +59,58 @@ public BadgeListDTO getUserBadges(BadgeQuery query) {
     ).collect(Collectors.toList())).build();
   }
 
+  public RankingDTO getCategoryRankings(String username, String categoryName, int page) {
+    AtomicBoolean addUser= new AtomicBoolean(true);
```

Comment:
`AtomicBoolean` should be used in multi-threaded environments, not as a general-purpose wrapper for a boolean value.